build-Index.ps1

```diff
@@ -1,12 +1,27 @@
 param(
-    [string]$title = $env:pagetext
+    [string]$title = $env:pageText,
```

Comment:
Made the pagetext environment variable camel case.
Added environment varialbes useCustomPath and customPath.